UPGRADE.md

```diff
@@ -2,6 +2,43 @@
 
 ## dev-develop
 
+### New definition mechanism for image-formats
+
+A new structure for the image-formats configuration files was introduced.
+For an explanation on how to define image-formats in the new way please
+refer to the documentation (TODO: link to documentation).
```

Comment:
Can't you already anticipate the link? From what I see in https://github.com/sulu/sulu-docs/pull/264 it hasn't changed.